### DISCUSSION TITLE
CTk Scrollbar error

### DISCUSSION THREAD

**Discussion by Storik4pro** [TomSchimansky/CustomTkinter#763] [Q&A] [answered]
I'm using this code, but ...
`scrollbar = CTkScrollbar(tabRed, command=canvas.yview)
 scrollbar.grid(padx=50, pady=20, row=0, column=2)`
but i have an error.
`self._button_color = ThemeManager.theme["CTkScrollbar"]["scrollbar_color"] if button_color is None else self._check_color_type(button_color)
KeyError: 'scrollbar_color'`
_Sorry for my English)_

**Comment by TomSchimansky** [comment, marked as answer]
https://github.com/TomSchimansky/CustomTkinter/issues/711

**Reply by Storik4pro** [reply]
Thanks)
